[deliver] undefined local variable or method `v' for Deliver UploadMetadata

<!-- Thanks for helping fastlane! Before you submit your issue, please make sure you followed our checklist and check the appropriate boxes by putting an x in the [ ]: [x] -->

### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

I tried to upload metadata to App Store, but got the following error:

```
bundler: failed to load command: fastlane (<path/to/app/>/ios/vendor/bundle/ruby/2.7.0/bin/fastlane)
NameError: [!] undefined local variable or method `v' for #<Deliver::UploadMetadata:0x00007fa077196b48>
```

##### Command executed

```
% make
% cat Makefile
upload_metadata:
	bundle exec fastlane deliver \
		--edit_live=true \
		--skip_binary_upload=true \
		--skip_screenshots=true \
		--skip_meta=false \
		--skip_app_version_update=true \
		--submit_for_review=false \
		--run_precheck_before_submit=false \
		--force
```

##### Complete output when running fastlane, including the stack trace and command used
<!-- 
You can use `--capture_output` as the last command line argument for many commands to get that collected for you. Otherwise, please do it manually.
Caution: The output of `--capture_output` could contain sensitive data such as application ids, certificate ids, or email addresses. Please make sure you double check the output and replace anything sensitive you don't wish to submit in the issue 
-->

<details>

```
% make upload_metadata
bundle exec fastlane deliver \
		--edit_live=true \
		--skip_binary_upload=true \
		--skip_screenshots=true \
		--skip_meta=false \
		--skip_app_version_update=true \
		--submit_for_review=false \
		--run_precheck_before_submit=false \
		--force
[✔] 🚀 
[13:50:37]: Login to App Store Connect (<redacted>)
[13:50:38]: Login successful

+--------------------------------------+------------------------+
|                    deliver 2.179.0 Summary                    |
+--------------------------------------+------------------------+
| edit_live                            | true                   |
| skip_binary_upload                   | true                   |
| skip_screenshots                     | true                   |
| skip_metadata                        | false                  |
| skip_app_version_update              | true                   |
| submit_for_review                    | false                  |
| run_precheck_before_submit           | false                  |
| force                                | true                   |
| screenshots_path                     | ./fastlane/screenshots |
| metadata_path                        | ./fastlane/metadata    |
| username                             | <redacted>             |
| app_identifier                       | <redacted>             |
| platform                             | ios                    |
| use_live_version                     | false                  |
| overwrite_screenshots                | false                  |
| reject_if_possible                   | false                  |
| phased_release                       | false                  |
| reset_ratings                        | false                  |
| team_id                              | <redacted>             |
| dev_portal_team_id                   | <redacted>             |
| precheck_default_rule_level          | warn                   |
| ignore_language_directory_validation | false                  |
| precheck_include_in_app_purchases    | true                   |
+--------------------------------------+------------------------+

[13:50:42]: Loading './fastlane/metadata/en-US/description.txt'...
[13:50:42]: Loading './fastlane/metadata/en-US/keywords.txt'...
[13:50:42]: Loading './fastlane/metadata/en-US/release_notes.txt'...
[13:50:42]: Loading './fastlane/metadata/en-US/support_url.txt'...
[13:50:42]: Loading './fastlane/metadata/en-US/name.txt'...
[13:50:42]: Loading './fastlane/metadata/en-US/subtitle.txt'...
[13:50:42]: Loading './fastlane/metadata/en-US/privacy_url.txt'...
[13:50:42]: Loading './fastlane/metadata/primary_category.txt'...
[13:50:42]: Loading './fastlane/metadata/secondary_category.txt'...

Looking for related GitHub issues on fastlane/fastlane...

➡️  Suddenly getting error: muninitialized constant Fastlane::Actions::SharedValues::KEYCHAIN_PATH 
    https://github.com/fastlane/fastlane/issues/18063 [closed] 5 💬
    30 Jan 2021

🔗  You can ⌘ + double-click on links to open them directly in your browser.
bundler: failed to load command: fastlane (<path/to/app/>/ios/vendor/bundle/ruby/2.7.0/bin/fastlane)
NameError: [!] undefined local variable or method `v' for #<Deliver::UploadMetadata:0x00007fa077196b48>
  <path/to/app>/ios/vendor/bundle/ruby/2.7.0/gems/fastlane-2.179.0/deliver/lib/deliver/upload_metadata.rb:100:in `upload'
  <path/to/app>/ios/vendor/bundle/ruby/2.7.0/gems/fastlane-2.179.0/deliver/lib/deliver/runner.rb:149:in `upload_metadata'
  <path/to/app>/ios/vendor/bundle/ruby/2.7.0/gems/fastlane-2.179.0/deliver/lib/deliver/runner.rb:58:in `run'
  <path/to/app>/ios/vendor/bundle/ruby/2.7.0/gems/fastlane-2.179.0/deliver/lib/deliver/commands_generator.rb:81:in `block (2 levels) in run'
  <path/to/app>/ios/vendor/bundle/ruby/2.7.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:178:in `call'
  <path/to/app>/ios/vendor/bundle/ruby/2.7.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:153:in `run'
  <path/to/app>/ios/vendor/bundle/ruby/2.7.0/gems/commander-fastlane-4.4.6/lib/commander/runner.rb:476:in `run_active_command'
  <path/to/app>/ios/vendor/bundle/ruby/2.7.0/gems/fastlane-2.179.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:76:in `run!'
  <path/to/app>/ios/vendor/bundle/ruby/2.7.0/gems/commander-fastlane-4.4.6/lib/commander/delegates.rb:15:in `run!'
  <path/to/app>/ios/vendor/bundle/ruby/2.7.0/gems/fastlane-2.179.0/deliver/lib/deliver/commands_generator.rb:186:in `run'
  <path/to/app>/ios/vendor/bundle/ruby/2.7.0/gems/fastlane-2.179.0/deliver/lib/deliver/commands_generator.rb:17:in `start'
  <path/to/app>/ios/vendor/bundle/ruby/2.7.0/gems/fastlane-2.179.0/fastlane/lib/fastlane/cli_tools_distributor.rb:114:in `take_off'
  <path/to/app>/ios/vendor/bundle/ruby/2.7.0/gems/fastlane-2.179.0/bin/fastlane:23:in `<top (required)>'
  <path/to/app>/ios/vendor/bundle/ruby/2.7.0/bin/fastlane:23:in `load'
  <path/to/app>/ios/vendor/bundle/ruby/2.7.0/bin/fastlane:23:in `<top (required)>'
make: *** [upload_metadata] Error 1
```

</details>

### Environment

<!-- 
Please run `fastlane env` and copy the output below. This will help us help you.
If you used the `--capture_output` option, please remove this block as it is already included there. 
-->

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                                                  |
| --------------------------- | -------------------------------------------------------------------------------------- |
| OS                          | 10.15.7                                                                                |
| Ruby                        | 2.7.2                                                                                  |
| Bundler?                    | true                                                                                   |
| Git                         | git version 2.29.2                                                                     |
| Installation Source         | \<path/to/app>/ios/vendor/bundle/ruby/2.7.0/bin/fastlane                               |
| Host                        | Mac OS X 10.15.7 (19H524)                                                              |
| Ruby Lib Dir                | /usr/local/Cellar/ruby@2.7/2.7.2/lib                                                   |
| OpenSSL Version             | OpenSSL 1.1.1i  8 Dec 2020                                                             |
| Is contained                | false                                                                                  |
| Is homebrew                 | false                                                                                  |
| Is installed via Fabric.app | false                                                                                  |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                                            |
| Xcode Version               | 12.4                                                                                   |


### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |


### fastlane files:

**No Fastfile found**


<details><summary>`./fastlane/Appfile`</summary>

```ruby
app_identifier("<redacted>")
apple_id("<redacted>")

itc_team_id("<redacted>")
team_id("<redacted>")

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.179.0 | ✅ Up-To-Date  |


### Loaded fastlane plugins:

**No plugins Loaded**


<details><summary><b>Loaded gems</b></summary>

| Gem                           | Version      |
| ----------------------------- | ------------ |
| did_you_mean                  | 1.4.0        |
| bundler                       | 2.1.4        |
| uri                           | 0.10.0       |
| rake                          | 13.0.3       |
| CFPropertyList                | 3.0.3        |
| public_suffix                 | 4.0.6        |
| addressable                   | 2.7.0        |
| artifactory                   | 3.0.15       |
| atomos                        | 0.1.3        |
| aws-eventstream               | 1.1.1        |
| aws-partitions                | 1.436.0      |
| aws-sigv4                     | 1.2.3        |
| jmespath                      | 1.4.0        |
| aws-sdk-core                  | 3.113.0      |
| aws-sdk-kms                   | 1.43.0       |
| aws-sdk-s3                    | 1.93.0       |
| babosa                        | 1.0.4        |
| claide                        | 1.0.3        |
| colored                       | 1.2          |
| colored2                      | 3.1.2        |
| highline                      | 1.7.10       |
| commander-fastlane            | 4.4.6        |
| declarative                   | 0.0.20       |
| declarative-option            | 0.1.0        |
| digest-crc                    | 0.6.3        |
| unf_ext                       | 0.0.7.7      |
| unf                           | 0.1.4        |
| domain_name                   | 0.5.20190701 |
| dotenv                        | 2.7.6        |
| emoji_regex                   | 3.2.2        |
| excon                         | 0.79.0       |
| faraday-net_http              | 1.0.1        |
| multipart-post                | 2.0.0        |
| ruby2_keywords                | 0.0.4        |
| faraday                       | 1.3.0        |
| http-cookie                   | 1.0.3        |
| faraday-cookie_jar            | 0.0.7        |
| faraday_middleware            | 1.0.0        |
| fastimage                     | 2.2.3        |
| gh_inspector                  | 1.1.3        |
| jwt                           | 2.2.2        |
| memoist                       | 0.16.2       |
| multi_json                    | 1.15.0       |
| os                            | 1.1.1        |
| signet                        | 0.15.0       |
| googleauth                    | 0.16.0       |
| httpclient                    | 2.8.3        |
| mini_mime                     | 1.0.3        |
| uber                          | 0.1.0        |
| representable                 | 3.0.4        |
| retriable                     | 3.1.2        |
| google-api-client             | 0.38.0       |
| rexml                         | 3.2.4        |
| webrick                       | 1.7.0        |
| google-apis-core              | 0.3.0        |
| google-apis-iamcredentials_v1 | 0.2.0        |
| google-apis-storage_v1        | 0.3.0        |
| google-cloud-env              | 1.5.0        |
| google-cloud-errors           | 1.1.0        |
| google-cloud-core             | 1.6.0        |
| google-cloud-storage          | 1.31.0       |
| json                          | 2.5.1        |
| mini_magick                   | 4.11.0       |
| naturally                     | 2.2.1        |
| plist                         | 3.6.0        |
| rubyzip                       | 2.3.0        |
| security                      | 0.1.3        |
| simctl                        | 1.6.8        |
| slack-notifier                | 2.3.2        |
| terminal-notifier             | 2.0.0        |
| unicode-display_width         | 1.7.0        |
| terminal-table                | 1.8.0        |
| tty-screen                    | 0.8.1        |
| tty-cursor                    | 0.7.1        |
| tty-spinner                   | 0.9.3        |
| word_wrap                     | 1.0.0        |
| nanaimo                       | 0.3.0        |
| xcodeproj                     | 1.19.0       |
| rouge                         | 2.0.7        |
| xcpretty                      | 0.3.0        |
| xcpretty-travis-formatter     | 1.0.1        |
</details>


*generated on:* **2021-03-27**
</details>

--- a/deliver/lib/deliver/upload_metadata.rb
+++ b/deliver/lib/deliver/upload_metadata.rb
@@ -97,7 +97,7 @@ module Deliver
         localised_options = LOCALISED_LIVE_VALUES
         non_localised_options = NON_LOCALISED_LIVE_VALUES
 
-        if v.nil?
+        if version.nil?
           UI.message("Couldn't find live version, editing the current version on App Store Connect instead")
           version = fetch_edit_app_store_version(app, platform)
           # we don't want to update the localised_options and non_localised_options

